jpo-ode-core/src/main/java/us/dot/its/jpo/ode/dds/CASClient.java

```diff
@@ -1,243 +0,0 @@
-/*******************************************************************************
- * Copyright 2018 572682
- * 
- * Licensed under the Apache License, Version 2.0 (the "License"); you may not
- * use this file except in compliance with the License.  You may obtain a copy
- * of the License at
- * 
- *   http://www.apache.org/licenses/LICENSE-2.0
- * 
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
- * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
- * License for the specific language governing permissions and limitations under
- * the License.
- ******************************************************************************/
-package us.dot.its.jpo.ode.dds;
-
-import java.net.URI;
-import java.util.Collections;
-import java.util.Map;
-import java.util.concurrent.ConcurrentHashMap;
-import java.util.regex.Matcher;
-import java.util.regex.Pattern;
-
-import javax.net.ssl.SSLContext;
-import javax.ws.rs.core.Response.Status;
-
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-import us.dot.its.jpo.ode.wrapper.HttpClientFactory;
-import us.dot.its.jpo.ode.wrapper.HttpClientFactory.HttpClient;
-import us.dot.its.jpo.ode.wrapper.HttpClientFactory.HttpException;
-import us.dot.its.jpo.ode.wrapper.HttpClientFactory.HttpResponse;
-
-public class CASClient {

```

Comment:
Why do we remove all the jpo-ode-core DDS components?

Reply:
The ODE previously used the DDS/WebSocket functionality to make deposits to the SDX. However, since the SDX no longer accepts deposits through this method, the DDS components are no longer needed.